src/main/java/org/opentestsystem/ap/ibms/ItemBulkUpdateApi.java

```diff
@@ -93,4 +93,10 @@ public ItemBulkResponse updateItemAccessibility(@PathVariable String section,
         request.setSection(section);
         return svc.updateItemAccessibility(request);
     }
+
+    @PatchMapping("/association")
```

Comment:
Interesting. I'm curious, what is the philosophy around using PATCH?

Reply:
When updating an item's property (one property in a request) we have been using PATCH.  For example, when an item's workflow status changes we use patch.  Previous to this feature that was the only case TIMS was updating a single property for an item.  All other cases of editing an item are multiple properties, something we call a section.  Section changes are PUT. 

I used PATCH here because we are updating one property.  Though the request is for one or more items, so that is a bit different.  I did wonder if given that this is more of a PUT.

Reply:
Makes sense, thanks.

In a previous life, i tried the same thing (using PATCH when updating single fields in resources) and got voted down by the rest of the team.